lib/ansible/modules/cloud/azure/azure_rm_loadbalancer.py

```diff
@@ -816,130 +827,26 @@ def create_or_update_load_balancer(self, param):
         try:
             poller = self.network_client.load_balancers.create_or_update(self.resource_group, self.name, param)
             new_lb = self.get_poller_result(poller)
-            return load_balancer_to_dict(new_lb)
+            return new_lb.as_dict()
         except CloudError as exc:
             self.fail("Error creating or updating load balancer {0} - {1}".format(self.name, str(exc)))
 
 
-def load_balancer_to_dict(load_balancer):
-    """Seralialize a LoadBalancer object to a dict"""
-    if not load_balancer:
-        return dict()
-
-    result = dict(
-        id=load_balancer.id,
-        name=load_balancer.name,
-        location=load_balancer.location,
-        sku=load_balancer.sku.name,
-        tags=load_balancer.tags,
-        provisioning_state=load_balancer.provisioning_state,
-        etag=load_balancer.etag,
-        frontend_ip_configurations=[],
-        backend_address_pools=[],
-        load_balancing_rules=[],
-        probes=[],
-        inbound_nat_rules=[],
-        inbound_nat_pools=[],
-        outbound_nat_rules=[]
-    )
+def compare_arrays(old, new):
+    old = old or []
+    new = new or []
+
+    oldd = {}
+    for item in old:
```

Comment:
add a helper for repeat operation?

Reply:
I will move it to common as a last step of this pr, I use the same comparison function in appgateway actually. so when you approve everything else, I will do it, ac ci takes long time with changes in common....

Reply:
or do you mean array_to_dict() helper?